src/components/interface/editing/EditDropdown.jsx

```diff
@@ -1,6 +1,7 @@
 import React from 'react';
 
 import Menu from '@material-ui/core/Menu';
+import PropTypes from 'prop-types';
```

Comment:
Let's keep this import above material-ui imports and put it on line 2 with the import React components.

We typically import in groups: React specific, material UI, then components we've made.